packages/esm-ward-app/src/ward-workspace/create-admission-encounter/create-admission-encounter-action-button.extension.tsx

```diff
@@ -0,0 +1,42 @@
+import { ActionMenuButton, AddIcon, launchWorkspace } from '@openmrs/esm-framework';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { type CreateAdmissionEncounterWorkspaceProps } from './create-admission-encounter.workspace';
+
+function CreateAdmissionRequestActionButton() {
+  const { t } = useTranslation();
+  const [searchTerm, setSearchTerm] = useState<string>('');
```

Comment:
Do we actually need this? We don't seem to do anything with it other than update the workspace props?

Reply:
We can file it as potential future work.